app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -175,6 +175,21 @@ def alerts_client
       with_provider_connection(&:alerts)
     end
 
+    # server ops
+    def shutdown_middleware_server(ems_ref, params)
+      timeout = params[:timeout] || 0
+      run_generic_operation(:Shutdown, ems_ref, :restart => false, :timeout => timeout)
```

Comment:
@Jiri-Kremser - is there a default value for :restart? I see it is false here and true below.